src/app/auth.service.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/filter';
+import * as auth0 from 'auth0-js';
+
+@Injectable()
+export class AuthService {
+
+  auth0 = new auth0.WebAuth({
+    clientID: 'YdXJArBHl5cZiJP9Fjp4aCv2wd4LsYBg',
```

Comment:
Do we want this information stored as environment variables? Can always change this later. No need to worry about this atm.

Reply:
It's okay to have the clientID public, outside users will not be able to retrieve/configure anything in regards to that client ID without actually logging onto the Auth0 profile.